ios/AirshipReactNative.swift

```diff
@@ -328,10 +328,10 @@ public extension AirshipReactNative {
 // Actions
 @objc
 public extension AirshipReactNative {
-    func actionsRun(actionName: String, actionValue: Any?) async throws-> Any? {
+    func actionsRun(action: [String: Any]) async throws-> Any? {
         return try await AirshipProxy.shared.action.runAction(
-            actionName,
-            value: try AirshipJSON.wrap(actionValue)
+            action["_name"] as! String,
+            value: action["_value"] is NSNull ? nil : try AirshipJSON.wrap(action["_value"])
```

Comment:
I had to make sure I pass nil if it's an NSNull.
Otherwise the JSON wrapping fails: the NSNull Object don't go through the guard else, is that normal @crow ?
```
public static func wrap(_ value: Any?, encoder: JSONEncoder = AirshipJSON.defaultEncoder) throws -> AirshipJSON {
    guard let value = value else {
        return .null
    }
    .
    .
    .
}
```